static/js/biomedical/disease/drug_table.tsx

```diff
@@ -1,37 +1,54 @@
 import React from "react";
 
 import { GRAPH_BROWSER_REDIRECT } from "../bio_charts_utils";
-import { CompoundDiseaseTreatmentData } from "./chart";
-
-export interface DrugTableProps {
-  data: CompoundDiseaseTreatmentData[];
+import {
+  CompoundDiseaseContraindicationData,
+  CompoundDiseaseTreatmentData,
+} from "./chart";
+export interface DrugTreatmentTableColumn {
+  // the column id
+  id: string;
+  // the column display name or header
+  name: string;
 }
-
-export function DrugTable(props: DrugTableProps): JSX.Element {
+export interface DrugTreatmentTableProps {
+  // stores the column id and column name
+  // retains the order of the columns and column id matches the key of data
```

Comment:
nit: ... and column ids should match the keys of the objects in the data array

Reply:
Added that!